src/App.js

```diff
@@ -1,21 +1,21 @@
-/* 
-Start here and then work through the 
-PostsContainer components and the CommentSection Components. 
-Not all files need code added. 
-Look at each file to see where you need to pass props or add code 
-*/
-import React from "react";
+
+import React,{useState} from "react";
 import "./App.css";
-// import the PostsPage and SearchBar and add them to the App
-// import dummydata
+import PostsPage from "./components/PostsContainer/PostsPage";
+import SearchBar from "./components/SearchBar/SearchBarContainer";
+import dummyData from "./dummy-data";
+import CommentSection from "./components/CommentSection/CommentSectionContainer";
+
 
 const App = () => {
-  // set up state for the dummy data and pass to your PostsPage
+  const [data] = useState(dummyData);
```

Comment:
be sure to get in the habit of creating the setState function when you create a state